app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -3557,20 +3533,6 @@ class BrowserTabViewModelTest {
         }
     }
 
-    @Test
```

Comment:
Tested elsewhere, in https://github.com/duckduckgo/Android/pull/3582/files#diff-86ed09c4f784467e8f157d920ab5dbfcedbf2995ff42181622cf763407052d50